services/src/main/java/io/trello/trelloapp/TrelloCloneApp.java

```diff
@@ -10,4 +13,13 @@ public static void main(String[] args) {
 		SpringApplication.run(TrelloCloneApp.class, args);
 	}
 
+  @Bean
+	public WebMvcConfigurer corsConfigurer() {
+		return new WebMvcConfigurer() {
+			@Override
+			public void addCorsMappings(CorsRegistry registry) {
+				registry.addMapping("/**").allowedOrigins("http://localhost:4200");
```

Comment:
can u take this from application.properties

Reply:
moved to application.properties